File: ASCOM.Utilities/ASCOM.Utilities/AlpacaSupport/JSON Response Classes/AlpacaConfiguredDevicesResponse.vb
Imports System.Collections.Generic

Friend Class AlpacaConfiguredDevicesResponse
    Inherits RestResponseBase

    Public Sub New()
        Value = New ConfiguredDevice() {}
    End Sub

    Public Sub New(ByVal clientTransactionID As UInteger, ByVal transactionID As UInteger, ByVal value As ConfiguredDevice())
        ServerTransactionID = transactionID
        MyBase.ClientTransactionID = clientTransactionID
        Me.Value = value
    End Sub

    Public Property Value As ConfiguredDevice()
End Class

File: ASCOM.Utilities/ASCOM.Utilities/AlpacaSupport/JSON Response Classes/AlpacaConfiguredDevicesResponse.cs
using System.Collections.Generic;

namespace ASCOM.Utilities
{

    internal class AlpacaConfiguredDevicesResponse : RestResponseBase
    {

        public AlpacaConfiguredDevicesResponse()
        {
            Value = new List<ConfiguredDevice>();
        }

        public AlpacaConfiguredDevicesResponse(uint clientTransactionID, uint transactionID, List<ConfiguredDevice> value)
        {
            ServerTransactionID = transactionID;
            ClientTransactionID = clientTransactionID;
            Value = value;
        }

        public List<ConfiguredDevice> Value { get; set; }
    }
}